app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_names = command.split(" ")
```

Comment:
`" "` is default value for split
```suggestion
    file_names = command.split()
```